### PR TITLE
Update use-group-policy.md

### PR DESCRIPTION
**Pull request by ShrCaJesmo**
Fixes the table (looks like tables don't work right within Note blurbs) and adds a powershell snippet to unescape the ID since that's a bit easier than doing repeated replaces.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: OneDrive/use-group-policy.md [hunk 1]
```diff
@@ -494,14 +494,19 @@ To find the library ID, sign in as a global or SharePoint admin in Office 365, b
...
+|---- |-------|
+| %2D |   -   |
+| %7B |   {   |
+| %7D |   }   |
+| %3A |   :   |
+| %2F |   /   |
+| %2E |   .   |
+
+> [!NOTE]
+> Alternatively, in powershell you can run the following, replacing "Copied String" with the library ID above:
```
Comment (JohanFreelancer9) [inline OneDrive/use-group-policy.md]:
```suggestion
> Alternatively, you can run the following in PowerShell, replacing "Copied String" with the library ID above:
```

**Comment by ShrCaJesmo** [comment]
Review's been merged

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**